gpii/node_modules/ontologyHandler/test/node/OntologyHandlerTests.js

```diff
@@ -79,6 +79,159 @@ https://github.com/gpii/universal/LICENSE.txt
         ]
     };
 
+    var floydWarshallTestsLength = 18;
+    var floydWarshallTests = [
+        {
+            name: "Initial 4 vertice, 3 edges",
```

Comment:
Sorry - given it is not a word I got confused about what word it is meant to be :)
Current state of the branch is still showing it as "vertice" tho